src/AccessibilityInsights.Extensions.AzureDevOps/ConfigurationControl.xaml.cs

```diff
@@ -31,6 +31,7 @@ public ConfigurationControl()
         {
             InitializeComponent();
             DataContext = VMConfig;
+            serverTreeview.ItemsSource = null;
```

Comment:
This clears the error, but I'm not sure it's the correct fix. The  warning refers to a non-existent `ConfigurationViewModel.projects` property, then the `ChangeStates` property sets `serverTreeview.ItemsSource = projects`, where `projects` is on `ConfigurationControl` instead of the ViewModel. Do you think it would make sense to move `ConfigurationControl.projects` to `ConfigurationViewModel.projects` ?

(As a side note, the property should really be PascalCased, so `Projects`)

Reply:
> Do you think it would make sense to move ConfigurationControl.projects to ConfigurationViewModel.projects

Yup, that makes sense to me, I'll implement that